docs/source/deployment/CORS_Headers.rst

```diff
@@ -0,0 +1,73 @@
+========================================
+Add CORS Headers to server configuration
+========================================
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Overview
+========
+
+Cross-origin resource sharing (CORS) is the mechanism for resource sharing between servers located in different domains.
```

Comment:
Still too generic, where does Motech come into all this? Why would I want to share any resources? By the way, what resources are we talking about? Do I need to this if the UI is separate?